js/followUnfollowUser/updateFollowButton.mjs

```diff
@@ -0,0 +1,17 @@
+import { isUserFollowed } from "./isUserFollowed.mjs";
+import { loggedInUser } from "./index.mjs";
+
+export function updateFollowButton(currentFollowing) {
+  const btn = document.querySelectorAll(".follow-user");
+  const btnVariable = btn.forEach((e) => {
+    const users = e.id;
+    if (isUserFollowed(users, currentFollowing) == true) {
```

Comment:
Use triple equals when comparing values in JavaScript. [MDN - Equality comparisons and sameness](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness)

You could also check for a truthy value rather than specifically checking with `=== true` or `=== false`.
```js
const myVariable = true;

// these are the same
if (myVariable === true) {}
if (myVariable) {}

// these are the same
if (myVariable === false) {}
if (!myVariable) {}
```